### PR TITLE
Feature/edit profile fe

### PR DESCRIPTION
**Pull request by C-Bandstra**
### Type of change made:

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Styling (Styling to new or existing files)
- [ ] Testing

### Detailed Description

Add edit profile component

Button click renders the edit profile component

edit email input updates components state

Added id to the user data in the EditProfile component

When a user edits their email, the global store and local storage are updated

Data base accepts new updated changes

### Why is this change required? What problem does it solve?

This will allow the user to update their info

### Was any previously written code changed?

no

### Directory modified:

- [] insert file names here

### CODE REVIEW

File: code/api/package.json [hunk 1]
```diff
@@ -42,7 +42,7 @@
     "multer": "1.4.2",
     "mysql2": "2.1.0",
     "pg": "^8.2.1",
-    "sequelize": "^5.21.6",
+    "sequelize": "^5.22.3",
```
Comment (mikez321) [inline code/api/package.json:45]: Gettin' some back end in the front end! Nice!

File: code/web/src/modules/user/EditProfile.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+import { updateUser } from './api/actions'
+
+class EditProfile extends Component {
+  constructor(props) {
+    super(props)
+      this.state = {
+        error: '',
+        isLoading: false,
+        user: {
+          id: this.props.user.details.id,
```
Comment (mikez321) [inline code/web/src/modules/user/EditProfile.js:13]: This was the key!

**Review by mikez321 — APPROVED** [review]
Looking good, and really glad we were able to get this working with an ID not with something that can change like an email!

File: code/web/src/modules/user/EditProfile.js (hunk 2 above)
Comment (maddyrandle) [inline code/web/src/modules/user/EditProfile.js:13]: Nice job finding and grabbing this id!

**Comment by jklich151** [comment]
Worked together to get functionality for finding a user by ID.

**Review by maddyrandle — APPROVED** [review]
Great job!

**Pull request merged**